<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PyPdfEditor</class>
 <widget class="QMainWindow" name="PyPdfEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pdf Editor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolTip
{
    color: #b1b1b1;
    border: 1px solid black;
    background-color:rgb(42,42,42);
    padding: 1px;
    border-radius: 3px;
    opacity: 190;
}

QWidget
{
    color: #b1b1b1;
    background-color: #323232;
    border: 0px;
}

QLabel
{
    color: #b1b1b1;
    min-height: 1px;
    background-color: rgba(50,50,50,0);
}

QListWidget:item:hover
{
    background-color: rgb(50,60,74);
}

QListWidget:item:selected
{
    background-color: rgb(70,80,94);
    color: #FFFFFF;
    border: 2px solid rgb(255,255,255);

}

QWidget:item
{
    alternate-background-color: '#2a2b2c';
}

QMenuBar::item
{
    background: transparent;
}

QMenuBar::item:selected
{
    background: transparent;
    border: 1px solid #48628e;
}

QMenuBar::item:pressed
{
    background: #444;
    border: 1px solid #000;
    background-color: #48628e;
    margin-bottom:-1px;
    padding-bottom:1px;
}

QMenu
{
    border: 1px solid #000;
    padding: 1px 2px 1px 2px;
}

QMenu::item
{
    padding: 2px 20px 2px 20px;
}

QMenu::item:selected
{
    background-color: #48628e;
    color: #000000;
}

QWidget:disabled
{
    color: #404040;
    background-color: #323232;
}

QLineEdit
{
    background-color: #282828;
    padding: 1px;
    border-style: solid;
    border: 1px solid #1e1e1e;
    border-radius: 2;
}

QLineEdit:!enabled {
    color: #B1B1B1;
    background-color: transparent;
}

QPushButton {
    color: #b4b4b4;
    border: 1px solid #2a2a2a;
    border-radius: 9px;
    padding: 5px;
    background: #292929;
    min-width: 12px;
    min-height: 12px;
}

QPushButton:default {
    border : 1px solid #3c5176;
}

QPushButton:hover {
    color: #FFFFFF;
    border: 2px solid #48628e;
    padding: 0px;
}

QPushButton:pressed {
    background: #3d3d3d;
}

QComboBox
{
    background-color: rgb(36,36,36);
    border: 1px solid #1e1e1e;
}

QComboBox:item
{
    background-color: transparent;
    border-style: solid;
    border: 1px solid #1e1e1e;
    border-radius: 2;
}

QComboBox:hover
{
    border: 2px solid #48628e;
}

QComboBox:on
{
    selection-background-color: #ffaa00;
}

QComboBox QAbstractItemView
{
    selection-background-color: #48628e;
}

QComboBox::drop-down
{
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;

    border-left-width: 1px;
    border-left-color: rgb(35,35,35);
    border-left-style: solid;
    background-color: rgb(93,93,93);
 }

 QComboBox::down-arrow, QSpinBox::down-arrow, QTimeEdit::down-arrow, QDateEdit::down-arrow
{
    width: 16px;
    height: 14px;
}


QGroupBox:focus
{
border: 2px solid QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);
}

QTextEdit#publishNotes:focus
{
    border: 2px solid #48628e;
}

QPlainTextEdit
{
    background-color: #242424;
}

QHeaderView::section
{
    background-color: #444444;
    border-right: 1px solid #2c2c2c;
    border-left: 0px;
    border-top: 0px;
    border-bottom: 0px;
}

QMainWindow::separator
{
    background-color: QLinearGradient(x1:0, y1:0, x2:0, y2:1, stop:0 #161616, stop: 0.5 #151515, stop: 0.6 #212121, stop:1 #343434);
    color: white;
    padding-left: 4px;
    border: 1px solid #4c4c4c;
    spacing: 0px; /* spacing between items in the tool bar */
}

QMainWindow::separator:hover
{

    background-color: QLinearGradient(x1:0, y1:0, x2:0, y2:1, stop:0 #d7801a, stop:0.5 #b56c17 stop:1 #ffa02f);
    color: white;
    padding-left: 4px;
    border: 1px solid #6c6c6c;
    spacing: 0px; /* spacing between items in the tool bar */
}

QMenu::separator
{
    height: 2px;
    background-color: QLinearGradient(x1:0, y1:0, x2:0, y2:1, stop:0 #161616, stop: 0.5 #151515, stop: 0.6 #212121, stop:1 #343434);
    color: white;
    padding-left: 4px;
    margin-left: 10px;
    margin-right: 5px;
}

QProgressBar
{
    height: 22px;
    border: 0px solid #1e1e1e;;
    border-radius: 2px;
    text-align: center;
    background-color: transparent
}

QProgressBar::chunk
{
    background-color: #48628e;
    width: 2.15px;
    margin: 0px;
}

QCheckBox
{
    color: rgb(240,240,240)
}

QRadioButton::indicator:hover
{
    border: 1px solid #6889c1;
}

QStatusBar {
    background-color: #2e2e2e;
}

QStatusBar::item {
    border: none;
}

QAbstractScrollArea
{
    border-radius: 4px;
    border: 1px solid #3A3939;
}

QScrollBar:horizontal
{
    height: 15px;
    margin: 0px 11px 0px 11px;
    border: 1px solid #3A3939;
    border-radius: 6px;
    background-color: QLinearGradient( x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 #302F2F, stop: 1 #484846);
}

QScrollBar::handle:horizontal
{
    background-color: QLinearGradient( x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 #605F5F, stop: 1 #787876);
    min-width: 5px;
    border-radius: 5px;
}

QScrollBar::add-line:horizontal
{
    width: 10px;
    height: 10px;
    subcontrol-position: right;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:horizontal
{
    height: 10px;
    width: 10px;
    subcontrol-position: left;
    subcontrol-origin: margin;
}

QScrollBar::add-line:horizontal:hover,QScrollBar::add-line:horizontal:on
{
    height: 10px;
    width: 10px;
    subcontrol-position: right;
    subcontrol-origin: margin;
}


QScrollBar::sub-line:horizontal:hover, QScrollBar::sub-line:horizontal:on
{
    height: 10px;
    width: 10px;
    subcontrol-position: left;
    subcontrol-origin: margin;
}

QScrollBar::up-arrow:horizontal, QScrollBar::down-arrow:horizontal
{
    background: none;
}


QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal
{
    background: none;
}

QScrollBar:vertical
{
    background-color: QLinearGradient( x1: 1, y1: 0, x2: 0, y2: 0, stop: 0 #302F2F, stop: 1 #484846);
    width: 12px;
    margin: 11px 0 11px 0;
    border: 1px solid #3A3939;
    border-radius: 3px;
}

QScrollBar::handle:vertical
{
    background-color: QLinearGradient( x1: 1, y1: 0, x2: 0, y2: 0, stop: 0 #101010, stop: 1 #262626);
    min-height: 5px;
    border-radius: 5px;
}

QScrollBar::sub-line:vertical
{
    height: 10px;
    width: 10px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}

QScrollBar::add-line:vertical
{

    height: 10px;
    width: 10px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical:hover,QScrollBar::sub-line:vertical:on
{

    height: 10px;
    width: 10px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}


QScrollBar::add-line:vertical:hover, QScrollBar::add-line:vertical:on
{
    height: 10px;
    width: 10px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;
}

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical
{
    background: none;
}


QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical
{
    background: none;
}

QListWidget
{
    background-color: rgba(58, 58, 58, 255);
    alternate-background-color: rgb(36,46,60);
    padding: 0px;
}

QListWidget::item
{
    background-color: transparent;
}

QToolBar
{
    background-color: transparent ;
    color: transparent ;
}

QToolButton:hover
{
    color: #5d81bd;
}

QSplitter::handle {
    border: 1px solid #333333;
    height: 2px;
    background-color: #101010;
    border-radius: 4px;
}

QSplitter::handle:pressed {
    background-color: #6187ac;
}

QTreeWidget
{
    background-color: #151515;
    alternate-background-color: #2a2a2a;
    border: 0px;
}
</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <widget class="QFrame" name="main_frame">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-30</y>
      <width>733</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="main_layout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>3</number>
     </property>
     <item row="3" column="0" colspan="2">
      <widget class="QFrame" name="generate_frame">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="output_label">
          <property name="text">
           <string>Output</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="output_line_edit">
          <property name="text">
           <string>Folder for generated files</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="browse_button">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="generate_button">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>Generate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="close_button">
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QFrame" name="pdf_view_frame">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>3</number>
       </property>
       <property name="midLineWidth">
        <number>3</number>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QFrame" name="top_frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="output_frame">
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="input_frame">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>3</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_new"/>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
    <addaction name="separator"/>
    <addaction name="action_add_pdfs"/>
    <addaction name="action_remove_pdfs"/>
    <addaction name="action_merge_pdfs"/>
    <addaction name="action_split_pdfs"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_new_document"/>
    <addaction name="action_remove"/>
    <addaction name="separator"/>
    <addaction name="action_prefs"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_edit"/>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
  <action name="action_undo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo (Ctrl + Z)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_redo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo (Ctrl + Y)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="action_new">
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>Create New Setup (Ctrl + N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open Setup (Ctrl + O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save Setup (Ctrl + S)</string>
   </property>
  </action>
  <action name="action_add_pdfs">
   <property name="text">
    <string>Add PDFs</string>
   </property>
   <property name="toolTip">
    <string>Add PDFs (Ins)</string>
   </property>
   <property name="shortcut">
    <string>Ins</string>
   </property>
  </action>
  <action name="action_remove_pdfs">
   <property name="text">
    <string>Remove PDFs</string>
   </property>
   <property name="toolTip">
    <string>Remove PDFs (Ctrl + D)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit (Ctrl + Q)</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="action_merge_pdfs">
   <property name="text">
    <string>Merge PDFs</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="action_split_pdfs">
   <property name="text">
    <string>Split PDFs</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="action_new_document">
   <property name="text">
    <string>Add New Documet</string>
   </property>
  </action>
  <action name="action_remove">
   <property name="text">
    <string>Remove Page/Document</string>
   </property>
  </action>
  <action name="action_remove_page">
   <property name="text">
    <string>Remove Page</string>
   </property>
  </action>
  <action name="action_prefs">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
